schemaless/test_mapblklot_generator.py

```diff
@@ -0,0 +1,29 @@
+# Lint as: python3
+from schemaless.mapblklot_generator import MapblklotGenerator
+
+
+def test_behavior_before_instantiation():
+    assert(MapblklotGenerator.get_instance() is None)
+
+
+def test_one_instance():
+    MapblklotGenerator('data/assessor/2020-02-18-parcels.csv.xz')
+
+    instance = MapblklotGenerator.get_instance()
+    assert(instance is not None)
+    # Make sure that subsequent calls to get the instance returns the same
+    # underlying instance.
+    assert(instance == MapblklotGenerator.get_instance())
```

Comment:
nit: I think you should do `instance is ...`, because `is` ensures it's the same object, rather than just equivalent.